cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/SystemArguments.java

```diff
@@ -56,6 +57,9 @@
   public static final String TRANSACTION_TIMEOUT = "system.data.tx.timeout";
   public static final String METRICS_ENABLED = "system.metrics.enabled";
   public static final String METRICS_CONTEXT_TASK_INCLUDED = "system.metrics.context.task.included";
+  private static final String YARN_MAX_APP_ATTEMPTS = "system.yarn.max.app.attempts";
```

Comment:
I don't see this property being documented.

Reply:
This PR don't have doc changes. These are system runtime arguments, will have a separate PR to add it to doc (probably not in 4.3).